atmoz-sftp.yaml

```diff
@@ -0,0 +1,80 @@
+#nolint:valid-pipeline-git-checkout-commit,valid-pipeline-git-checkout-tag
+package:
+  name: atmoz-sftp
+  version: 0.0.0_git20250226
+  epoch: 0
+  description: Secure File Transfer Protocol (SFTP) server based on OpenSSH, from the https://github.com/atmoz/sftp
```

Comment:
This description still sounds like it is the official SFTP program. 

Maybe something like `Various scripts from https://github.com/atmoz/sftp to help run SFTP in a container` ?